mail/mail_test.go

```diff
@@ -0,0 +1,15 @@
+package mail
+
+import (
+	"testing"
+)
+
+func TestParseTemplate(t *testing.T) {
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("The code did not panic.")
+		}
+	}()
+	parseTemplate("./mail_template.html", nil)
```

Comment:
I think, this test always succeeds, even if you remove `./mail_template.html` ;-)